app/views/accounts/_sidebar_show.html.haml

```diff
@@ -25,24 +24,25 @@
   %div= render "shared/address_show", asset: @account, type: 'shipping', title: :shipping_address
 
   .caption #{t :account_summary}
-  %li
-    %dt= pipeline != 0.0 ? number_to_currency(pipeline, precision: 0) : t(:n_a)
-    %tt #{t :pipeline}:
-  %li
-    %dt= won != 0.0 ? number_to_currency(won, precision: 0) : t(:n_a)
-    %tt #{t :won}:
-  %li
-    %dt= lost != 0.0 ? number_to_currency(lost, precision: 0) : t(:n_a)
-    %tt #{t :lost}:
-  %li
-    %dt= @account.category ? t(@account.category) : t(:other)
-    %tt #{t :category}:
-  %li
-    %span{style: "float:right;"}= stars_for(@account)
-    %tt #{t :rating}:
-  %li
-    %dt= @account.assigned_to ? truncate(@account.assignee.full_name, length: 16) : nil
-    %tt #{t :assigned_to}:
+  %dl
+    %li
+      %dt= pipeline != 0.0 ? number_to_currency(pipeline, precision: 0) : t(:n_a)
```

Comment:
can we use unless pipeline.zero? instead. might make the code more readable and ruby-esque

Reply:
Yes that's a good idea.

Reply:
I broadly just lifted and shifted existing code (here its mostly just adding a dl element; so it can be stuck into a flexbox model); but how about we put that into the rubocop rules/apply that globally?

Reply:
I've captured this one in #929 so we can do wider cleanup